graphiti_core/search/search_utils.py

```diff
@@ -164,38 +156,30 @@ async def edge_fulltext_search(
     filter_query, filter_params = edge_search_filter_query_constructor(search_filter)
 
     query = (
-        get_relationships_query('edge_name_and_fact', db_type=driver.provider)
+        get_relationships_query('edge_name_and_fact', provider=driver.provider)
         + """
         YIELD relationship AS rel, score
```

Comment:
There seems to be a bug here. `rel` is not used anywhere in the rest of the query, which means `e` below is a search over all edges, no?

Reply:
Added a fix by matching the `uuid`.

Reply:
thanks for catching!